src/lib/commands.rs

```diff
@@ -235,20 +241,23 @@ where
         syscall!(w.trussed.delete(key));
     }
 
-    w.send_to_output({
-        CommandSignResponse {
-            inhash: req.hash,
-            signature,
-        }
-    });
+    send_to_output(
+        {
+            CommandSignResponse {
+                inhash: req.hash,
+                signature,
+            }
+        },
```

Comment:
Are these brackets necessary?

```suggestion
        CommandSignResponse {
            inhash: req.hash,
            signature,
        }
```